etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -124,7 +124,7 @@ spec:
           value: {{ .Values.pachd.storage.local.hostPath | default $randHostPath }}pachd
         {{- end }}
         - name: WORKER_IMAGE
-          value: "{{ .Values.global.image.registry }}{{ .Values.pachd.worker.image.repository }}:{{ default .Chart.AppVersion .Values.pachd.image.tag }}"
+          value: "{{ .Values.global.image.registry }}{{ .Values.pachd.worker.image.repository }}:{{ default .Chart.AppVersion (or .Values.pachd.worker.image.tag .Values.pachd.image.tag) }}"
```

Comment:
what's the difference between WORKER_IMAGE and WORKER_SIDECAR_IMAGE on line 136?

Reply:
the worker is the worker, the sidecar is pachd.

Reply:
Huh, I thought the sidecar was the pachd running in the storage container. Are we purposely not adding the value to the values.yaml file, because we don't want users to change it?

Reply:
the sidecar is the pachd running in the storage container.  the environment variable sets the version of pachd.

users do not need to change this value; they should always use the same tag for pachd and the worker.  only in local dev would we have different versions.  (the image tag is based on the content of the image, not when it was built.)